rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/vendor/Sw360VendorService.java

```diff
@@ -186,4 +186,20 @@ public Set<Release> getAllReleaseList(String vendorId) throws TException {
         Set<Release> releases = componentsClient.getReleasesByVendorId(vendorId);
         return releases;
     }
+
+    public RequestStatus mergeVendors(String vendorTargetId, String vendorSourceId, Vendor vendorSelection, User user) throws TException {
+        VendorService.Iface sw360VendorClient = getThriftVendorClient();
+        RequestStatus requestStatus;
+        requestStatus =  sw360VendorClient.mergeVendors(vendorTargetId, vendorSourceId, vendorSelection, user);
+
+        if (requestStatus == RequestStatus.IN_USE) {
+            throw new HttpMessageNotReadableException("Vendor already in use.");
```

Comment:
This status is returned when source/target vendor has an MR open. Message should reflect the same.